### DISCUSSION TITLE
V2 Chakra UI Select dropdown doesn't align correctly in browser mobile simulator

### DISCUSSION THREAD

**Discussion by p19ky** [chakra-ui/chakra-ui#9050] [General] [answered]
### Description

When I open the <Select /> component dropdown, the dropdown is aligned weirdly around the input. It should align normaly under it.

### Link to Reproduction

https://v2.chakra-ui.com/docs/components/select/usage

### Steps to reproduce

1. open developer console
2. toggle device toolbar and select iphone 14 pro max from the dimensions dropdown from the top center of the browser.
3. open any select dropdown on the v2 chakra ui usage docs for select component
4. you will see the dropdown of the select weirdly aligned.

<img width="1512" alt="Screenshot 2024-10-31 at 15 43 38" src="https://github.com/user-attachments/assets/32af253e-eaa8-43f8-8601-ab3216e39e9f">

### Chakra UI Version

2.10.3

### Browser

Arc Browser (Chromium based)

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by p19ky** [comment]
<img width="1512" alt="Screenshot 2024-10-31 at 15 43 38" src="https://github.com/user-attachments/assets/0ce75ea0-ba51-479f-adb5-8a96a580c3f7">

**Comment by segunadebayo** [comment, marked as answer]
v2 is in maintenance. We are not shipping new v2 work. Stay on 2.8.x or migrate to v3.
